extensions/2.0/Khronos/KHR_materials_clearcoat/README.md

```diff
@@ -61,29 +61,58 @@ All implementations should use the same calculations for the BRDF inputs. Implem
 |**clearcoatRoughnessFactor**      | `number`                                                                        | The clearcoat layer roughness.         | No, default: `0.0`   |
 |**clearcoatRoughnessTexture**     | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo)             | The clearcoat layer roughness texture. | No                   |
 |**clearcoatNormalTexture**        | [`normalTextureInfo`](/specification/2.0/README.md#reference-normaltextureinfo) | The clearcoat normal map texture.      | No                   |
-  
+
+
+The clearcoat effect is modeled via a microfacet-based BRDF. The BRDF is layered on top of the glTF 2.0 Metallic-Roughness material, including emission and all extensions.
+
+```
+# from glTF 2.0 Metallic-Roughness material (core)
+material = mix(dielectric_brdf, metal_brdf, metallic) + emission
+
+# coating BRDF
+clearcoat_brdf = specular_brdf((clearcoatRoughnessFactor * clearcoatRoughnessTexture.g)^2)
+
+# layer coating on top of base material
+coated_material = fresnel_coat(
+  layer = clearcoat_brdf,
+  base = material,
+  ior = 1.5,
+  weight = clearcoatTexture.r * clearcoatFactor,
+  normal = clearcoatNormalTexture)
+```
+
+The `fresnel_coat` operator adds a BRDF as a layer on top of another BSDF according to `weight` and a Fresnel term. The layer is weighted with `weight*fresnel(ior)`. The base is weighted with `1-(weight*fresnel(ior)`. `normal` is a float3 vector that affects the top layer, but not the base.
```

Comment:
Minor: `1-(weight*fresnel(ior)` is missing closing parenthesis.

Reply:
According to the implementation below, layer is weighted with just `weight`.

Reply:
Fixed.